src/tools/m65common.c

```diff
@@ -881,8 +888,10 @@ int fetch_ram(unsigned long address, unsigned int count, unsigned char* buffer)
     while (addr != end_addr) {
       snprintf(next_addr_str, 8192, "\n:%08X:", (unsigned int)addr);
       int b = serialport_read(fd, &read_buff[ofs], 8192 - ofs);
-      if (b < 0)
+      if (b <= 0)
```

Comment:
can go back to b < 0 now that you have removed the else path, since it makes no sense to set b=0 when b==0

Reply:
Aah, I'd left the <= there if per-chance I want to uncomment debug stuff. If it becomes '<', the debug printf below triggers a lot of un-needed garbage.

In the past, I'd usually clean out all my debug code prior to the pull request, but as I've found myself debugging in this realm of the code quite often, I'm thinking I'd better leave a bit of my extra debug stuff around, just in-case there's a next time :)